Header header
string group
geometry_msgs/PointStamped point
float64[] positions
